pages/sec_Quest/secQSchool.html

```diff
@@ -6,100 +6,133 @@
+      <a href="#">Classifications and Definitions</a>
+      <a href="#">Confirmation Page</a>
   </div>
-  <div class="col-10 sq-cover bg-light">
-    <h5 class="container">Annual Education Census Questionnaire Secondary School 2020-21 Academic Year</h5>
-    <form action="" method="post" autocomplete="off" class="sq-form">
-        <fieldset class="sq-school-fieldset">
+
+  
+  <div class="col-10 sq-form">  
+    <h3 class="sq-section-heading" style="font-style: italic; padding: 5px;">Annual Education Census Questionnaire 
+      Secondary School 2020-21 Academic Year</h3>
+    <form action="" method="post" autocomplete="off" >
+        <fieldset class="sq-form-fieldset">
             <legend>School Information</legend>
             <label for="school">Name of School: &nbsp;</label>
-            <input type="text" name="school" id="school"><br>
+            <input type="text" name="school" id="school" class="sq-form-input"><br>
             <label for="estdate">Date Established: &nbsp;</label>
-            <input type="date" name="estdate" id="estdate"><br>
+            <input type="date" name="estdate" id="estdate" class="sq-form-input"><br>
             <label for="district">Education District: &nbsp;</label>
-            <input type="text" name="district" id="district"><br>
+            <input type="text" name="district" id="district" class="sq-form-input"><br>
             <label for="principal">Principal: &nbsp;</label>
-            <input type="text" name="principal" id="principal"><br>
+            <input type="text" name="principal" id="principal"  class="sq-form-input"><br>
             <label for="address">School Address: &nbsp;</label><br>
-            <textarea name="schooladdress" id="schooladdress" cols="40" rows="5">
+            <textarea name="schooladdress" id="schooladdress" cols="40" rows="5" class="sq-form-input">
             </textarea><br>
             <label for="phone">Telephone: &nbsp;</label>
-            <input type="tel" name="phone" id="phone"><br>
+            <input type="tel" name="phone" id="phone" class="sq-form-input"><br>
             <label for="fax">Fax: &nbsp;</label>
-            <input type="tel" name="fax" id="fax"><br>
+            <input type="tel" name="fax" id="fax" class="sq-form-input"><br>
             <label for="email">School Email: &nbsp;</label>
-            <input type="email" name="email" id="email"><br>
+            <input type="email" name="email" id="email" class="sq-form-input"><br>
             <label for="pemail">Principal Email: &nbsp;</label>
-            <input type="email" name="pemail" id="pemail"><br>
-
-
+            <input type="email" name="pemail" id="pemail" class="sq-form-input"><br>
         </fieldset>
-        <fieldset class="sq-school-fieldset">
+
+        <fieldset class="sq-form-fieldset">
             <legend>Management Structure</legend>
+            <label for="mgt">Management Structure: &nbsp;</label>
```

Comment:
Use of label without id to link to the input

Reply:
Updated the suggested changes.